templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -59,10 +59,6 @@ contents:
           
         securityContext:
           priviledged: true
```

Comment:
This looks like a typo....

Reply:
This is a typo.  Can you open a second PR to remove this (since it's not needed apparently?)